brands/shop/car.json

```diff
@@ -164,6 +164,7 @@
     "tags": {
       "brand": "Jaguar",
       "brand:wikidata": "Q26742231",
+      "brand:wikipedia": "en:Jaguar Cars",
```

Comment:
0dd5929e5753422f29f7f503707b153458eb9b30 removes this tag, because the “Jaguar Cars” article is about the formerly independent manufacturer, whereas Q26742231 is about the marque that continues to be used by Jaguar Land Rover.